background.js

```diff
@@ -1,121 +1,124 @@
-chrome.commands.onCommand.addListener(function (command) {
-  if (command.match("account-[0-9]") ) {
+chrome.runtime.onMessage.addListener(function (command) {
     chrome.tabs.query({ active: true, lastFocusedWindow: true }, tabs => {
     
       var current_url = tabs[0].url;
       var account_num = command.substring(command.length - 1);
       var update_url_regex = null;
       var update_acc = null;
 
-      // Gmail
-      const gmail_regex = "https:\/\/mail.google.com\/mail\/u\/[0-9].*";
-      if (current_url.match(gmail_regex)) {
-        update_url_regex = RegExp("u\/[0-9]");
-        update_acc = "u/" + account_num
-      }
+        // Gmail
+        const gmail_regex = "https:\/\/mail.google.com\/mail\/u\/[0-9].*";
```

Comment:
There has been an unnecessary change in indentation for the whole block, please revert his, the git diffs generated would spoil the commit history.